Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml

```diff
@@ -603,6 +605,15 @@ Item {
 
         if (leaderPosition === Enums.LeaderPosition.Automatic) {
 
+            // Move leader vertically if vertical position isn't optimal
```

Comment:
@JamesMBallard Last time when I tried checking vertical first, I was hitting other issues with right and left positioning not working ok. Have you tested all 8 positions?